### DISCUSSION TITLE
Has anyone integrated Apache ECharts 5.0 with Superset?

### DISCUSSION THREAD

**Discussion by Beknazar19** [apache/superset#13047] [Q&A / Help]
For example, I need to build a waterfall, but it is not in the templates

**Comment by srinify** [comment]
Hi @Beknazar19 you might like this post by @amitmiran137 he made a waterfall chart!

https://medium.com/nmc-techblog/apache-superset-manage-custom-viz-plugins-in-production-9fde1a708e55
